code/stable/glassbr/src/cpp/InputParameters.cpp

```diff
@@ -205,10 +205,8 @@ void InputParameters::input_constraints() {
         std::cout << ", but is expected to be ";
         std::cout << "between ";
         std::cout << 0.1;
-        std::cout << " (d_min)";
```

Comment:
While here we output a variable name in "implementation" `Stage`, in other areas, we have the flexibility of printing any general `CodeExpr`.